contracts/payloads/IGP86/PayloadIGP86.sol

```diff
@@ -315,79 +353,30 @@ contract PayloadIGP86 is PayloadIGPMain {
         }
     }
 
-    function updateDexBaseLimits(uint256 dexId) internal {
-
+    function updateDexBaseLimits(uint256 dexId, uint256 maxSupplyShares, uint256 maxBorrowShares) internal {
         address dexAddress = getDexAddress(dexId);
+        if (dexAddress == address(0)) return;
+
+        (address AddressTokenA, address AddressTokenB) = getDexTokens(dexAddress);
 
-        address AddressTokenA = IFluidDex(dexAddress).tokenA();
-        address AddressTokenB = IFluidDex(dexAddress).tokenB();
+        uint256 maxSupplySharesInUSD = getRawAmount(address(0), maxSupplyShares, 0, true);
```

Comment:
I meant we already input the actual USD amount of shares value. Otherwise we have to resolve shares value in IGP, which complicates things 

so the input param to this method is `maxSupplySharesInUSD` and same for borrow. no call to getRawAmount will be needed

Reply:
i think the issue is the actually the $ value slightly varies as well imo?
so resolving the shares to dollar value with some function in the payload itself makes more sense to me

Reply:
that is true for all limits we set. we always hardcode the dollar value for a token for the getRawAmount method, and over the 4 days it always changes. 

for the effect, it doesn't matter if we hardcode the value of a share or if we input the total $ amount right away

just to explain, if you'd want fully correct pricing, then we'd have to, for each dex, for each supply and borrow separately:
- read max shares from storage
- get token0 and token1 per share
- from a linked vault or hardcode referenced oracle read the $ value per share

Reply:
the share value can change indeed e.g. for USDC / ETH vault, ETH max borrow right now is actually getting too close to max shares 
![image](https://github.com/user-attachments/assets/7d063009-c583-4f69-9a25-a17a58f79bd3)

let's set max borrow at 1.5 actually instead of 1.1

Reply:
for peg assets the effect of that price volatility is less problematic as the share value moves in sync mostly

Reply:
yes correct